samples/gui/deploy.sh

```diff
@@ -27,23 +27,21 @@ spec:
         imagePullPolicy: Always
         envFrom:
         - configMapRef:
-            name: m4d-config
+            name: gui-config
```

Comment:
I would suggest calling it m4dgui-config.  When searching the  namespace there are often other guis

Reply:
Renamed.